EIPS/eip-4910.md

```diff
@@ -0,0 +1,1111 @@
+---
+eip: 4910
+title: Royalty Bearing NFTs
+description: Extension of the ERC-721 standard for NFTs to correctly define, process, and pay (hierarchical) onchain royalties from NFT sales, going beyond EIP-2981.
+author: Andreas Freund (@Therecanbeonlyone1969) 
+discussions-to: https://ethereum-magicians.org/t/royalty-bearing-nfts/8453
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-14
+requires: 721, 165
+---
+
+## Abstract
+The proposal directly connects NFTs and royalties in a smart contract architecture extending the [ERC-721](./eip-721.md) standard, with the aim of precluding central authorities from manipulating or circumventing payments to those who are legally entitled to them.
+
+The proposal builds upon the [OpenZeppelin Smart Contract Toolbox](https://github.com/OpenZeppelin/openzeppelin-contracts) architecture, and extends it to include royalty account management (CRUD), royalty balance and payments management, simple trading capabilities -- Listing/De-Listing/Buying -- and capabilities to trace trading on exchanges. The royalty management capabilities allow for hierarchical royalty structures, referred to herein as royalty trees, to be established by logically connecting a "parent" NFT to its "children", and recursively enabling NFT "children" to have more children. 
+
+## Motivation
+The management of royalties is an age-old problem characterized by complex contracts, opaque management, plenty of cheating and fraud. 
+
+The above is especially true for a hierarchy of royalties, where one or more assets is derived from an original asset such as a print from an original painting, or a song is used in the creation of another song, or distribution rights and compensation are managed through a series of affiliates. 
+
+In the example below, the artist who created the original is eligible to receive proceeds from every sale, and resale, of a print. 
+
+![Fig1](/assets/eip-4910/eip-4910-print-families.png)
+
+
+The basic concept for hierarchical royalties utilizing the above "ancestry concept" is demonstrated in the figure below.
+
+![Fig2](/assets/eip-4910/eip-4910-royalties.png)
+
+
+In order to solve for the complicated inheritance problem, this proposal breaks down the recursive problem of the hierarchy tree of depth N into N separate problems, one for each layer. This allows us to traverse the tree from its lowest level upwards to its root most efficiently.
+
+This affords creators, and the distributors of art derived from the original, the opportunity to achieve passive income from the creative process, enhancing the value of an NFT, since it now not only has intrinsic value but also comes with an attached cash flow.
+
+## Specification Outline
```

Comment:
Not a valid section header as per EIP-1

```suggestion
## Specification

### Outline
```

Then bump the other sections in by one level.

Reply:
ok. thank you

Reply:
done